clis/teliod/src/main.rs

```diff
@@ -167,6 +392,11 @@ async fn daemon_event_loop(config: TeliodDaemonConfig) -> Result<(), TeliodError
                     Ok(ClientCmd::HelloWorld{name}) => {
                         info!("Hello {}", name);
                     }
+                    Ok(ClientCmd::GetMeshmap) => {
+                        if let Err(e) = tx.try_send(TelioTaskCmd::GetMeshmap) {
```

Comment:
When trying this I only get `Command executed successfully`but not a mesnet map.
Also how can I tell that the device was successfully registered? Unfortunately I don't see it when getting the meshnet map manually from the API

Reply:
This is removed now. It was there as a placeholder before.

Reply:
Right, and status reporting is part of a different ticket 😅